### PR TITLE
Updated thumbprint for the renewed cert

### LINKED ISSUE #3939: Automation certificate "AzureADIdentityDivisionTestAgentCert" will be renewed on Feb 8th
Automation certificate "AzureADIdentityDivisionTestAgentCert" in Azure Key Vault "BUILDAUTOMATION" will soon be renewed (Feb 8th) 

Azure Key Vault will renew your (Not configured) certificate on February 8, 2023 19:25 UTC. If applicable, please ensure your certificate registration is up-to-date. For more details, see the link below.

**Account information**
Subscription ID: c1686c51-b717-4fe0-9af3-24a20a41fb0c
Key Vault name: BUILDAUTOMATION
Certificate name: AzureADIdentityDivisionTestAgentCert
Certificate thumbprint: 4E87313FD450985A10BC0F14A292859F2DCD6CD3

This is used in [test automation](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/4ad4790397c7099e34c6ff5de768a43475d16e22/tests/Microsoft.Identity.Test.LabInfrastructure/LabAuthenticationHelper.cs#L19)

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes #3939

**Changes proposed in this request**
Updated thumbprint for test

**Testing**
DevOps tests

**Performance impact**
None

**Documentation**
- [x] All relevant documentation is updated.

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]

**Review by trwalke — APPROVED** [review]
LGTM

**Review by pmaytak — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static HashSet<string> s_scope
         public static readonly string[] s_graphScopes = new[] { "user.read" };
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
-        public const string AutomationTestThumbprint = "4E87313FD450985A10BC0F14A292859F2DCD6CD3";
+        public const string AutomationTestThumbprint = "444B697D869032F29F9A162D711AF3E2791AD748";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Test.Common/TestConstants.cs:34]: Is there a reason we don't get the certs by name?

**Review by neha-bhargava — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs (hunk 1 above)
Comment (bgavrilMS) [reply tests/Microsoft.Identity.Test.Common/TestConstants.cs:34]: It's coming in a separate commit.

**Review by bgavrilMS — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs (hunk 1 above)
Comment (SameerK-MSFT) [reply tests/Microsoft.Identity.Test.Common/TestConstants.cs:34]: Due to the time limitation, we are doing it as two-step process. This is the first one. The second is in draft: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3950

**Pull request closed without merging**